src/index.js

```diff
@@ -10,12 +10,15 @@ import Result from "./Pages/Result/Result";
 import Rules from "./Pages/Rules/Rules";
 import NavBar from "./Components/Navbar/Navbar";
 import Footer from "./Components/Footer/Footer";
+import { NameContextProvider } from "./Context/name-context";
 
 ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
       <CategoryContextProvider>
```

Comment:
are we yet to use category context ? I see that the import might be pending

Reply:
No, the import is not pending. All these files are being exported to the Home page where we need category to render cards of a particular category.